snippets/terraform/deployments/with-diagnostic-setting/main.tf

```diff
@@ -0,0 +1,59 @@
+terraform {
```

Comment:
this is a little confusing as the original example also has the notion of enabling diagnostics

Reply:
Maybe rename this directory to something else?

native logging? or something?

Reply:
I do not disagree it can be confusing with the original example, but I think the issue lies more with the original naming than the naming of this example. I'd prefer to keep these examples named this way as it is clear what the additional resource being created is.

Compromise on 'with-diagnostic-setting-logging'?